### DISCUSSION TITLE
[Q] Cell comment width calculation

### DISCUSSION THREAD

**Discussion by SyedSalmanNizami** [exceljs/exceljs#2409] [Q&A]
I have been trying to use cell Comment from exceljs but it is of a fixed width . Can't we introduce a feature for cell comment where we could define the width and height of each cell comment based on the content of it ? 

I guess it would be very helpful if this is implemented .

**Comment by SyedSalmanNizami** [comment]
Hi Team,

Do we have any updates yet ?

**Comment by chdh** [comment]
Duplicate of #1465?

**Comment by SyedSalmanNizami** [comment]
Yes . They both are the same .

**Comment by eminenceit** [comment]
any update on this issue please? This is an important feature!

**Comment by SyedSalmanNizami** [comment]
It has been an year almost now . Do we have any updates on this yet ?

**Comment by csgka1** [comment]
yes it's an important feature!  For now the only solution I've found is to add VBA script, however it's not a good solution.

**Comment by SyedSalmanNizami** [comment]
> yes it's an important feature! For now the only solution I've found is to add VBA script, however it's not a good solution.

Could you share that VBA code snippet ?

**Comment by csgka1** [comment]
> > yes it's an important feature! For now the only solution I've found is to add VBA script, however it's not a good solution.
> 
> Could you share that VBA code snippet ?

https://www.extendoffice.com/documents/excel/1572-excel-autosize-comments.html

Excel will prevent macro code from executing in the excel downloaded from Internet, so it doesn't match our requirement, but I hope it can help you.

**Comment by SyedSalmanNizami** [comment]
> > > yes it's an important feature! For now the only solution I've found is to add VBA script, however it's not a good solution.
> > 
> > 
> > Could you share that VBA code snippet ?
> 
> https://www.extendoffice.com/documents/excel/1572-excel-autosize-comments.html
> 
> Excel will prevent macro code from executing in the excel downloaded from Internet, so it doesn't match our requirement, but I hope it can help you.

Okay thanks . Will try this until they provide a solution 😀
